bot/code_review_bot/report/mail_builderrors.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import structlog
+
+from code_review_bot import taskcluster
+from code_review_bot.report.base import Reporter
+
+logger = structlog.get_logger(__name__)
+
+EMAIL_HEADER = """
+# Found {build_errors} build errors.
+
+Review Url: {review_url}
+
+"""
+
+
+class MailBuildErrorsReporter(Reporter):
```

Comment:
`BuildErrorsReporter` is a fine name, we are not doing Java Enterprise stuff :laughing: